### PR TITLE
Port to python3

### PR DESCRIPTION
**Pull request by justuswilhelm**
OK, Python 3.4 support works fine. It would be cool to have it test for Python 2.7 as well at some point. Are you planning to support Python 2 long term?

### CODE REVIEW

File: gitfs/utils/args.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# Copyright 2015 Justus Perlwitz
```
Comment (vtemian) [inline gitfs/utils/args.py]: The line added breaks the apache 2.0 license under which the code is published. Please check yourself http://www.apache.org/dev/apply-license.html#contributor-copyright

**Comment by vtemian** [comment]
Great work, just a short remark regarding the license. `Though committers retain copyright, Apache asks that they do not add copyright statements. See the policy for more details.`

**Comment by justuswilhelm** [comment]
Thanks! I will remove the copyright notes. Good catch :)

**Comment by vtemian** [comment]
We don't have any long term plans for supporting Python 2+. I hope that in 1 year to drop the Python 2 compatibility.

**Comment by justuswilhelm** [comment]
@vtemian I've removed all the copyright notes.

**Pull request merged**